libs/designsystem/src/lib/components/form-field/_form-field-inputs.shared.scss

```diff
@@ -1,9 +1,5 @@
 @use '~@kirbydesign/core/src/scss/utils';
-
-$form-field-input-font-family: var(--kirby-font-family);
-$form-field-input-line-height: 1.5;
-$form-field-input-padding: 1em;
-$form-field-label-height: 24px;
+@import '~@kirbydesign/core/src/scss/base/variables';
```

Comment:
`@kirbydesign/core/src/scss/base/variables` is already included in `@kirbydesign/core/src/scss/utils` so this line should be removed